app/frontend/src/details.js

```diff
@@ -18,9 +17,12 @@ function addDynamicSummaryTextForExpandedAndCollapsedDetailsTag(detailsElement)
     }
 }
 
-
 $( document ).ready(function() {
     $('details').each(function(_, details) {
         addDynamicSummaryTextForExpandedAndCollapsedDetailsTag(details);
     });
 });
+
```

Comment:
If you change to the code splitting imports as shown in my comment on application.js, you shouldn't need these as the `const`s are declared there.  It'll mean explicitly exporting the functions, though.